e2e/benchmarks/browserstack-benchmark/app.js

```diff
@@ -362,6 +363,12 @@ function setupHelpMessage() {
     help: 'runs GCP compatible version of benchmarking system',
     action: 'store_true'
   });
+  parser.add_argument('--weeklyRun', {
+    help: 'runs a part of models specified in the file everyday (the period ' +
+      'is a week), if --benchmarks is set',
+    default: false,
+    action: 'store_true'
+  });
```

Comment:
Nit: It might be nice to have a way to run a specific day's tests for debugging purposes. Maybe if you pass a day of the week to this flag, it can run that day's tests, but by default, it uses the current date.

Reply:
Great catch, done. Thank you!